src/components/user-signup.jsx

```diff
@@ -1,66 +1,136 @@
+                            type="email" 
+                            component="input" />
+                        </div>
+                      </div>
+                    </div>
 
-        <div className="form-group clearfix">
-          <label className="col-sm-12 control-label">
-            <FormattedMessage
-              id="shared.confirmPassword"
-              defaultMessage="Confirm Password"
-            />
-          </label>
-          <div className="col-sm-12">
-            <input type="text" htmlFor="" className="form-control" />
-          </div>
-        </div>
+                    <div className="form-group clearfix">
+                      <div className="row">
+                        <div className="col-sm-6">
 
-        <div className="form-group clearfix">
-          <div className="col-sm-12 text-center">
-            <a className="btn btn-default btn-lg btn-common" onClick={ this.props.toggle }>
-              <FormattedMessage
-                id="shared.login"
-                defaultMessage="Login"
-              />
-            </a>
-            <a className="btn btn-success btn-lg btn-common">
-              <FormattedMessage
-                id="shared.signUp"
-                defaultMessage="Sign Up"
-              />
-            </a>
-          </div>
-        </div>
-      </form>
+                          <label className="modal-form-label">
+                            <FormattedMessage
+                              id="shared.password"
+                              defaultMessage="Password"
+                            />
+                          </label>
+                          <div className="modal-form-input">
+                            <Field 
+                            className="form-input" 
+                            name="user[password]" 
+                            type="password" 
+                            component="input" />
+                          </div>
+                        </div>
+                        <div className="col-sm-6">
+                          <label className="modal-form-label">
+                            <FormattedMessage
+                              id="shared.confirmPassword"
+                              defaultMessage="Confirm Password"
+                            />
+                          </label>
+                          <div className="modal-form-input">
+                            <Field 
+                            className="form-input" 
+                            name="user[password_confirmation]" 
+                            type="password" 
+                            component="input" />
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                    <br />
+                    <div className="form-group clearfix">
+                      <div className="col-sm-12 text-center">
+                          <button className="button-primary">
```

Comment:
`left-align` this button. Just to keep it consistent with the submit button in login form modal.